src/consensus/mod.rs

```diff
@@ -27,6 +27,9 @@ pub mod encode;
 #[doc(hidden)]
 #[macro_use]
 pub mod endian;
+/// implements strict_encode for a few types
+pub mod strict_encode;
+pub use strict_encoding::Error;
```

Comment:
Why do we need `pub use` here?

Reply:
fixed in commit 1bd883f

Reply:
however not sure this should be merged to master as it might be too specific to farcaster node